### PR TITLE
More cleanup around the workers

### PR DESCRIPTION
**Pull request by Kailai-Wang**
### Context

This PR:
- removes heartbeat-timeout related test which is unused
- removes pallet-drop3 integration (the pallet itself is still kept)
- removes unused itp-types definitions

### CODE REVIEW

File: bitacross-worker/app-libs/stf/src/trusted_call.rs [hunk 1]
```diff
@@ -208,7 +210,8 @@ impl TrustedCallVerification for TrustedCallSigned {
 		payload.append(&mut mrenclave.encode());
 		payload.append(&mut shard.encode());
 
-		self.signature.verify(payload.as_slice(), self.call.sender_identity())
+		self.signature.verify(&blake2_256(&payload), self.call.sender_identity())
```
Comment (felixfaisal) [inline bitacross-worker/app-libs/stf/src/trusted_call.rs:213]: I would say we keep only one way of signing message, that is hashing it before we sign it. I always assumed the sign and verify methods did the hashing internally 😅

File: bitacross-worker/app-libs/stf/src/trusted_call.rs [hunk 2]
```diff
@@ -155,7 +156,8 @@ impl TrustedCallSigning<TrustedCallSigned> for TrustedCall {
 		payload.append(&mut mrenclave.encode());
 		payload.append(&mut shard.encode());
 
-		TrustedCallSigned { call: self.clone(), nonce, signature: pair.sign(payload.as_slice()) }
+		// use blake2_256 hash to shorten the payload - see `verify_signature` below
+		TrustedCallSigned { call: self.clone(), nonce, signature: pair.sign(&blake2_256(&payload)) }
```
Comment (felixfaisal) [inline bitacross-worker/app-libs/stf/src/trusted_call.rs:160]: The size of the signature remains same, I think we are replacing the computation of signing a large message with the computation of producing the hash for a large message. I do prefer hashing a message before signing it for consistency.

**Review by felixfaisal — APPROVED** [review]
LGTM, Just had a few comments regarding the payload signing.

File: bitacross-worker/app-libs/stf/src/trusted_call.rs (hunk 2 above)
Comment (felixfaisal) [reply bitacross-worker/app-libs/stf/src/trusted_call.rs:160]: Please ignore this, It was rhetorical

File: bitacross-worker/app-libs/stf/src/trusted_call.rs (hunk 1 above)
Comment (Kailai-Wang) [reply bitacross-worker/app-libs/stf/src/trusted_call.rs:213]: Yea this is changed in tee-worker already, just not in bc-worker.

**Review by kziemianek — APPROVED** [review]

**Pull request merged**